Week 0/Reverse String/reverse_string.swift

```diff
@@ -0,0 +1,14 @@
+// https://developer.apple.com/documentation/swift/array/2893281-swapat
```

Comment:
Esse link aqui é uma boa deixar! Porque ele é a documentação do método que você está usando no código. 

Só vou te pedir pra adicionar acima dele o link pro desafio do LeetCode também, pra ficar mais rápido de achar quando alguém for ver esse código 🤖